src/contracts/FlowToken.cdc

```diff
@@ -169,38 +170,34 @@ pub contract FlowToken: FungibleToken {
     //
     // The numbers are arbitrary.
     //
-    init() {
-        // Initialize the totalSupply field to the initial balance
-        self.totalSupply = 1000.0
+    init(adminAcct: AuthAccount) {
+        self.totalSupply = 0.0
```

Comment:
If we are delegating minting privileges to another account, I don't really think it makes to mint an initial supply into a vault that nobody can access.

Reply:
This is fine. As long as the correct initial supply of tokens get minted at genesis